ush/forecast_postdet.sh

```diff
@@ -191,6 +191,9 @@ EOF
     IEMS=${IEMS:-1}
   fi
 
+# NoahMP table
+  ${NLN} "${HOMEgfs}/parm/ufs/land/noahmptable.tbl" "${DATA}/noahmptable.tbl" 
```

Comment:
Please check first if this file is present, and if so, then link.
If it is not present, please abort with an appropriate message.